about.html

```diff
@@ -14,12 +14,21 @@
 
     <body>
```

Comment:
# Optional:
- [ ] About Mobile size:
- Your hamburger does not work when the webpage is in the About page in mobile size.
![Capture](https://user-images.githubusercontent.com/63972670/227234956-2f16f535-e158-4f7e-b863-b6373db256a3.JPG)
- I trust you will fix that because this project could land you a new job contract.